layout-management-services/src/main/java/org/exoplatform/layoutmanagement/rest/EntityBuilder.java

```diff
@@ -85,4 +91,32 @@ public static NodeLabelRestEntity toNodeLabelRestEntity(Map<Locale, State> nodeL
     nodeLabelRestEntity.setSupportedLanguages(supportedLanguages);
     return nodeLabelRestEntity;
   }
+
+  public static SiteRestEntity toSiteRestEntity(Site site, OrganizationService organizationService) {
+    if (site == null) {
+      return null;
+    }
+    SiteType siteType = Util.from(site.getType());
+    String displayName = site.getDisplayName();
+
+    if (StringUtils.isBlank(displayName) && SiteType.GROUP.equals(siteType)) {
+      try {
```

Comment:
No need to have OrganizationService as a global variable, just create it as a local variable when the site is of type Group